tutorials/notebooks/example_keras_pruning_mnist.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
Check that the notebook is defined correctly, because I'm not able to open it

Reply:
If you're referring to Colab, you will not be able to open it yet since it links to the tutorial in `main` (which still doesn't exist). But you can see the file's preview in github, and it looks ok.